### PR TITLE
Add deployment files

### PR DESCRIPTION
**Pull request by vic3r**
# Integrate deployment files
Add `yaml` file for Kubernetes deployment and dockerfile for generating image and container

## Description
This PR includes:
- The files required for Continuous Delivery stage
- These files can be used in almost any Service Provider which supports Kubernetes
- It was changed the way of injecting environment variables

## Closes ticket
None

### CODE REVIEW

File: config/env.py [hunk 1]
```diff
@@ -1,9 +1,11 @@
...
+load_dotenv()
 
-os.environ["ELASTICSEARCH_HOST"] = ''
-os.environ["ELASTICSEARCH_INDEX"] = ''
-os.environ["CORENLP_HOST"] = ''
-os.environ["LOCAL_DATA_PATH"] = ''
-os.environ["LOCAL_DOWNLOAD_PATH"] = ''
-os.environ["DOWJONES_APIKEY"] = ''
-os.environ["DOWJONES_SNAPSHOTID"] = ''
+os.environ.setdefault("ELASTICSEARCH_HOST", "")
```
Comment (cerritows) [inline config/env.py:5]: :uff: 👍

File: main.py [hunk 2]
```diff
@@ -170,4 +170,4 @@ def update_article_list(selected_companies, search_clicks, load_count_clicks, se
     return ArticleList.create_article_list_layout(article_list)
 
 if __name__ == '__main__':
-	app.run_server()
+	app.run_server(debug=True, host='0.0.0.0')
```
Comment (cerritows) [inline main.py]: Do you think we should leave it as `debug=True`?

File: requirements.txt [hunk 3]
```diff
@@ -17,6 +17,9 @@ Flask==1.1.2
 Flask-Compress==1.5.0
 future==0.18.2
 gast==0.3.3
+google-auth==1.21.1
```
Comment (cerritows) [inline requirements.txt]: What do we need Google Auth for?

**Review by cerritows — COMMENTED** [review]
👁️ o 👁️

File: requirements.txt (hunk 3 above)
Comment (vic3r) [reply requirements.txt]: Maybe I just messed up my venv 🤔

**Review by cerritows — APPROVED** [review]
👍

**Pull request merged**